tests/queries/0_stateless/03174_least_greatest_for_null.sql

```diff
@@ -0,0 +1,17 @@
+drop table if exists test1;
```

Comment:
Future readers will appreciate a one-line comment what the test is testing. (Asking because the test name could mean many things and the test doesn't have comments otherwise)